src/pages/OfferNew/Introduction/Sidebar/StartDate.tsx

```diff
@@ -50,27 +77,117 @@ const Value = styled.div`
   }
 `
 
-const TooltipWrapper = styled.div`
-  margin-left: 0.5rem;
+const HandleSwitchingWrapper = styled.div`
+  margin-top: 10px;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 1.25rem;
+`
+
+const HandleSwitchingLabel = styled.span`
+  font-size: 0.8rem;
+  line-height: 0.9rem;
+  font-weight: 500;
+  color: ${colorsV2.gray};
+  width: 50%;
 `
 
-export const StartDate: React.FC<Props> = ({ insuredAtOtherCompany }) => {
+export const StartDate: React.FC<Props> = ({
+  offerId,
+  startDate,
+  currentInsurer,
+}) => {
+  const getDefaultDateValue = () => {
+    if (startDate) {
+      return parse(startDate, 'yyyy-MM-dd', new Date())
+    }
+
+    if (currentInsurer) {
+      return null
+    }
+
+    return new Date()
+  }
+
+  const [datePickerOpen, setDatePickerOpen] = React.useState(false)
+  const [dateValue, setDateValue] = React.useState(getDefaultDateValue)
   const textKeys = useTextKeys()
+  const [setStartDate] = useStartDateMutation()
+  const [removeStartDate] = useRemoveStartDateMutation()
+
+  const getDateLabel = () => {
+    if (dateValue) {
+      if (isToday(dateValue)) {
+        return textKeys.SIDEBAR_STARTDATE_CELL_VALUE_NEW()
+      }
+
+      return format(dateValue, 'dd MMM yyyy')
+    }
+
+    return textKeys.SIDEBAR_STARTDATE_CELL_VALUE_SWITCHER()
+  }
+
+  const gqlDateFormat = 'yyyy-MM-dd'
+
   return (
-    <Wrapper>
-      <Label>
-        <Title>{textKeys.SIDEBAR_STARTDATE_CELL_LABEL()}</Title>
-        <SubTitle>{textKeys.SIDEBAR_STARTDATE_CELL_SUBLABEL()}</SubTitle>
-      </Label>
-
-      <Value>
-        {insuredAtOtherCompany
-          ? textKeys.SIDEBAR_STARTDATE_CELL_VALUE_SWITCHER()
-          : textKeys.SIDEBAR_STARTDATE_CELL_VALUE_NEW()}
-        <TooltipWrapper>
-          <Tooltip size="lg" body={textKeys.SIDEBAR_STARTDATE_TOOLTIP()} />
-        </TooltipWrapper>
-      </Value>
-    </Wrapper>
+    <>
+      <RowButton
+        datePickerOpen={datePickerOpen}
+        onClick={() => setDatePickerOpen(!datePickerOpen)}
+      >
+        <Label>
+          <Title>{textKeys.SIDEBAR_STARTDATE_CELL_LABEL()}</Title>
+          <SubTitle>{textKeys.SIDEBAR_STARTDATE_CELL_SUBLABEL()}</SubTitle>
+        </Label>
+        <Value>
+          <DateLabel>{getDateLabel()}</DateLabel>
```

Comment:
just an idea, shall we maybe pull this from the quote instead? In that way we could at least show the "actual" date if there'd be errors